testing/tests/test_emitted_events_util.py

```diff
@@ -17,21 +19,21 @@ class MyCharmEvents(CharmEvents):
 
 class MyCharm(CharmBase):
     META = {'name': 'mycharm'}
-    on = MyCharmEvents()
+    on = MyCharmEvents()  # type: ignore[assignment]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
More idiomatic?
```suggestion
    def __init__(self, framework: ops.Framework) -> None:
```
Need to import `ops` or `Framework` though.
Feel free to reject as out of scope.